ParquetClassLibrary/Sandbox/MapParent.cs

```diff
@@ -39,6 +39,9 @@ public abstract class MapParent
         /// <summary>Exit, spawn, and other special points on the map.</summary>
         protected readonly List<SpecialPoint> _specialPoints = new List<SpecialPoint>();
 
+        /// <summary>Floors and walkable terrain on the map.</summary>
```

Comment:
nit: Comment fix

Reply:
Shoot, I thought I got this one.  Thanks, will fix!